src/main/java/org/folio/circulation/domain/representations/ClaimItemReturnedProperties.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.circulation.domain.representations;
+
+public class ClaimItemReturnedProperties {
+
+  private ClaimItemReturnedProperties() {}
+
+  public static final String ITEM_CLAIMED_RETURNED_DATE = "itemClaimedReturnedDateTime";
+  public static final String COMMENT = "comment";
+}
```

Comment:
I think these properties can be defined with the request class, as that should be the only part of the system that needs them.